src/components/List.js

```diff
@@ -114,26 +151,31 @@ const List = ({ token }) => {
                 ?.toLowerCase()
                 ?.includes(searchInput.toLowerCase().trim()),
             )
-            ?.map((doc) => {
+            ?.map((sortedList) => {
               return (
-                <li key={doc.id}>
+                <li
+                  key={sortedList.id}
+                  className={backgroundColor(sortedList)}
+                  aria-label={backgroundColor(sortedList)}
```

Comment:
We're able to dynamically assign the `backgroundColor` and `aria-label` with the backgroundColor function. 🌈

Reply:
🦄